clip.py

```diff
@@ -99,6 +107,24 @@ def __init__(self, width, height):
         self.is_record = False
         self.outputsPorts = set()
         self.outputsPorts.add(Clip.DEFAULT_OUTPUT)
+        self.scenes = {}
```

Comment:
I'm not sure wether a "next scene/precious scene" binding would make any sense - after all the way it will be used on stage is in general not linear. You really want to be able to load scene 0 and then scene3 directly. But probably you're right with the order. I propose a binding similar to the one you have with the volume - learn a row of *n* midi-inputs that will be associated with scenes 1 to *n*